solr/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrExporter.java

```diff
@@ -241,7 +241,7 @@ public static void main(String[] args) {
             .argName("ZK_HOST")
             .type(String.class)
             .desc(
-                "Specify the ZooKeeper connection string when connecting to Solr in SolrCloud mode. If omitted both the -b parameter and the -z parameter, connect to http://localhost:8983/solr. For example 'localhost:2181/solr'.")
+                "Specify the ZooKeeper connection string when connecting to Solr in SolrCloud mode. If omitted both the -s parameter and the -z parameter, connect to http://localhost:8983/solr. For example 'localhost:2181/solr'.")
```

Comment:
One more though, we do have in `ToolBase.java` the `public OptionGroup getConnectionOptions` that defines these options...   Though to use that, maybe the SolrExporter would need to be a direct part of `SolrCLI`?